.github/auto-comment.yml

```diff
@@ -1,71 +0,0 @@
-pullRequestOpened: |
```

Comment:
Let's remove this on a different PR

Reply:
we can, though at this point its more spam than good

Reply:
for ourselves - if we move towards more community-driven, something like this would be useful. Since that should be happening soon, let's address this elsewhere (and probably defer the conversation to the community).